tools/common/StaticRegistrar.cs

```diff
@@ -3803,20 +3803,21 @@ void Generate (string header_path, string source_path)
 			
 			Specialize (sb);
 
-			methods.StringBuilder.AppendLine ("} /* extern \"C\" */");
+			methods.WriteLine ();
+			methods.AppendLine ();
+			methods.AppendLine(sb);
```

Comment:
Style: there should be a space before the opening parenthesis here (+ a few more cases further down).